blockball-bukkit-plugin/src/main/java/com/github/shynixn/blockball/bukkit/BlockBallDependencyInjectionBinder.kt

```diff
@@ -93,6 +97,19 @@ class BlockBallDependencyInjectionBinder(private val plugin: BlockBallPlugin) :
         bind(CommandService::class.java).to(CommandServiceImpl::class.java).`in`(Scopes.SINGLETON)
         bind(GameExecutionService::class.java).to(GameExecutionServiceImpl::class.java).`in`(Scopes.SINGLETON)
 
+        when {
+            // TODO Implement EntityRegistrationServiceImpl lower than 1.14
```

Comment:
Why is the TODO still present?

Reply:
Looks like I omitted the commit for this PR. I'll fix it

Reply:
[Checkout my latest commit.](https://github.com/LazoYoung/BlockBall/commit/b86683be4ab143a08375778b2687cda72e19e86f)